examples/hitl/rearrange_v2/config/lang_rearrange_spot_humanoid.yaml

```diff
@@ -0,0 +1,39 @@
+# @package _global_
+
+defaults:
+  - language_rearrange_multi_agent
+  - hitl_defaults
+  - _self_
+
+habitat:
+  # various config args to ensure the episode never ends
+  environment:
+    max_episode_steps: 0
+    iterator_options:
+      # For the demo, we want to showcase the episodes in the specified order
+      shuffle: False
```

Comment:
More attributes must be added here for the desired outcome.

See [this PR](https://github.com/facebookresearch/habitat-lab/pull/1935).